### PR TITLE
media-sound/jack: fix #298850, new EAPI and eclass

### PR DESCRIPTION
**Pull request by madmartin**
- fix bug https://bugs.gentoo.org/show_bug.cgi?id=298850
- EAPI 6
- replace eclass distutils with distutils-r1
- fix ebuild python module dependencies
- add proxy maintaintership
- add functional enhancement: add cddb server freedb.musicbrainz.org

Package-Manager: Portage-2.3.3, Repoman-2.3.1

replaces #4011

### CODE REVIEW

**Comment by madmartin** [comment]
@mgorny please take a look at this replaced PR

File: media-sound/jack/metadata.xml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+<maintainer type="person">
```
Comment (SoapGentoo) [inline media-sound/jack/metadata.xml]: in general, every tag level needs an indent. Leave this, I will fix it on merging

File: media-sound/jack/jack-3.1.1-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	dev-python/cddb-py[${PYTHON_USEDEP}]
+	dev-python/id3-py[${PYTHON_USEDEP}]
+	dev-python/pyid3lib[${PYTHON_USEDEP}]
+	dev-python/pyvorbis[${PYTHON_USEDEP}]
+	media-libs/flac
+	media-sound/lame
+	media-sound/cdparanoia"
+REQUIRED_USE=${PYTHON_REQUIRED_USE}
```
Comment (SoapGentoo) [inline media-sound/jack/jack-3.1.1-r2.ebuild]: the idiomatic place for `REQUIRED_USE` is directly after `IUSE`. Don't worry, I'll fix it up on merging

**Comment by SoapGentoo** [comment]
@madmartin please rebase on fresh master though, you still have a broken master

**Comment by madmartin** [comment]
@mgorny Hi, I updated my repository fork, now there is only one issue report from qa-bot. Is this a problem caused by me?

**Pull request closed without merging**